src/apps/components/AppFrame/AppFrame.tsx

```diff
@@ -100,7 +106,10 @@ export const AppFrame: React.FC<Props> = ({
       src={urlJoin(
         src,
         window.location.search,
-        `?domain=${shop.domain.host}&id=${appId}&locale=${locale}`,
+        `?${stringifyQs(
+          { domain: shop.domain.host, id: appId, ...params },
+          "comma",
+        )}`,
```

Comment:
Could you move that to a separate function? let's keep building URL logic in one place

Reply:
Updated. WDYT?

Reply:
how about moving this to `@saleor/apps/urls` ?

Reply:
@andrzejewsky updated